engine/table/src/main/java/io/deephaven/engine/util/PyCallableWrapper.java

```diff
@@ -103,7 +112,7 @@ public void setChunkSourceIndex(int chunkSourceIndex) {
 
         public int getChunkSourceIndex() {
             if (!resolved) {
-                throw new IllegalStateException("The column chunk argument hasn't been resolved " + this.columnName);
+                throw new IllegalStateException("The column chunk argument hasn't been resolved " + columnName);
```

Comment:
This should be re-worded. Maybe:
```suggestion
                throw new IllegalStateException("The column chunk argument for " + columnName + " hasn't been resolved");
```